src/tablib.egg-info/PKG-INFO

```diff
@@ -0,0 +1,464 @@
+Metadata-Version: 2.1
```

Comment:
Should the `*.egg-info` dir be Git ignored and removed from source control?

For example:

* https://github.com/github/gitignore/blob/21419e391a78f2487340b3b1240b988aaf15b54f/Python.gitignore#L25